gensim/summarization/keywords.py

```diff
@@ -186,11 +188,12 @@ def _format_results(_keywords, combined_keywords, split, scores):
         return [(word, _get_average_score(word, _keywords)) for word in combined_keywords]
     if split:
         return combined_keywords
-    return "\n".join(combined_keywords)
+    return to_utf8("\n".join(combined_keywords))
```

Comment:
We use utf8 only on output (serializing to files, external APIs etc). Inside gensim, we try to keep all text as unicode.